.github/ISSUE_TEMPLATE/new_scraper.md

```diff
@@ -0,0 +1,20 @@
+---
+name: New website scraper request
+about: Add support for a new recipe website
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+We always enjoy adding new websites to the list of [available `recipe-scrapers` scrapers](https://github.com/hhursev/recipe-scrapers#scrapers-available-for)!
+
+To help us out, please check that recipes on the website are public - we can't currently scrape recipes that require an account login - and add sample recipe URL(s) below:
+
+- https:// ...
+
+Can you write Python and would you like to help add the scraper yourself?  We'd be glad for your assistance!  We can provide you with guidance and code review in return.  If so, tick any of the relevant boxes below:
```

Comment:
Good question - to explain a bit, initially I thought about not having a third checkbox at all.  But then I wondered whether people might not fully read or comprehend the prompt, and instead be drawn straight to the 'interactive' options -- and so I figured it might be worth adding one to allow/try to clear up any misunderstandings there by providing an option.

Unfortunately markdown doesn't seem to support radio buttons as far as I know, so your example may indeed happen :)

I think in that case we'd probably want to ask the issue author when they might like to get started working on it, and based on that we could decide whether we'd be likely to implement it ourselves (based on availability and what the site/schema look like) or allow them to try it themselves.

If we ask that it might turn out it was an accidental double-selection, or we could discuss the timeline (without providing any guarantees of course).

(that's a lot of semantics/context to describe in the file -- so perhaps there's a better way to phrase the options and/or prompt to help with all that - ideas welcome!)

Reply:
NB: Also potentially relevant context: I do think that (at least as currently written), ticking boxes one and two is a valid combination.

Reply:
I agree that ticking boxes 1 and 2 at the same time. I'm not worried about this and I'm pretty sure this gets into deep lore about the English language. Not a big issue.